lib/messenger.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './price.rb'
+class Messenger
+  def initialize(client, order, to_from)
+    @client = client
+    @order = order
+    @to_from = to_from
+  end
+
+  def message
+    { from: @to_from[:from], to: @to_from[:to],
+      body: "Hi there, your order totalling £#{Price::format(@order.total)} " +
```

Comment:
I like this `Price::format` thing but perhaps it could also generate the '£'.  Then you could have different formatters for different (english speaking) regions of the world.

Reply:
That is a very good idea. I extracted this class after realising that I was a repeating myself across multiple classes. I think the addition of the currency symbols is a really good idea as it would make it more useful and abstract.